plugin.yaml

```diff
@@ -1781,6 +1781,9 @@ node_types:
         delete:
           implementation: awssdk.cloudify_awssdk.ec2.resources.instances.delete
           inputs: *operation_inputs
+        modify_instance_attribute:
```

Comment:
@EarthmanT This method will be triggered by user not as a part of the install lifecycle

Reply:
The plugin operation "modify_instance_attribute" will not be executed during install. A user would need to call it from execute_operation or from another custom workflow. However, I think we need to get in the habit of exposing operations like this. Since it helps with writing custom workflows and because we may have a custom install soon.